nik4.py

```diff
@@ -219,6 +231,9 @@ def add_fonts(path):
 	bbox = None
 	rotate = not options.norotate
 
+        if options.ozi and options.projection:
+                raise Exception('ozi map file output is only supported for Web Mercator (EPSG:3857). Please remove --projection.')
```

Comment:
Since `options.projection` has a default value, it is always set. Maybe do not set default and check for `None` here?

Reply:
fixed